src/express-tooling.ts

```diff
@@ -17,7 +17,7 @@ import { getTokenInfo as defaultGetTokenInfo } from './oauth-tooling';
 import {
   AuthenticationMiddlewareOptions,
   ExtendedRequest,
-  onAuthorizationFailedHandler,
+  onAuthorizationFailedHandler as onAuthorizationFailedHandlerType,
```

Comment:
I am not quite sure about the intent of this renaming, would the import type feature help here? 
 `import type { .... } from `

Reply:
I also do not see a reason to rename here